roles/openshift_manage_node/tasks/main.yml

```diff
@@ -19,10 +19,12 @@
     -n default
   register: omd_get_node
   until: omd_get_node.rc == 0
-  retries: 50
+  retries: 10
   delay: 5
   changed_when: false
   with_items: "{{ openshift_nodes }}"
+  when: hostvars[item].openshift.node.nodename is defined
+  ignore_errors: true
```

Comment:
I'm not sure I'm a fan of this approach, seems like it'll lead to other issues if we just allow all of these to fail.

What if before openshift_manage_node is applied we had some task that attempts to poll for the availability similar to above, and then if the retries is reached, we only run o_m_n against the set of hosts that are available from `oc get nodes`?